app/mailers/patient_mailer.rb

```diff
@@ -209,50 +219,18 @@ def closed_email(patient)
 
   private
 
-  def add_success_history(patient, parent)
-    comment = if patient == parent
-                "Sara Alert sent a report reminder to this monitoree via #{parent.preferred_contact_method}."
+  def add_success_history(patient)
+    comment = if patient.id == patient&.responder_id
+                "Sara Alert sent a report reminder to this monitoree via #{patient.preferred_contact_method}."
               else
-                "Sara Alert sent a report reminder to this monitoree's HoH via #{parent.preferred_contact_method}."
+                "Sara Alert sent a report reminder to this monitoree's HoH via #{patient.responder.preferred_contact_method}."
               end
-    History.report_reminder(patient: patient, comment: comment)
-  end
-
-  def add_fail_history_sms(patient)
-    comment = "Sara Alert attempted to send an SMS to #{patient.primary_telephone}, but the message could not be delivered."
-    History.report_reminder(patient: patient, comment: comment)
-  end
-
-  def add_fail_history_voice(patient)
-    comment = "Sara Alert attempted to call monitoree at #{patient.primary_telephone}, but the call could not be completed."
-    History.report_reminder(patient: patient, comment: comment)
+    History.report_reminder(patient: patient, comment: comment) unless patient&.preferred_contact_method.nil? && patient&.id == patient&.responder_id
```

Comment:
I put this check here (and other places report_reminders are created) as a response to this comment https://github.com/SaraAlert/SaraAlert/pull/690#discussion_r574796628. Whether this check is redundant is a good question, I need to think about why this check existed in the History model in the first place to determine if it can be removed outright in this PR.

Reply:
I would remove it personally, I don't think `add_success_history` will ever be called if they don't have a valid preferred contact method.